stack.yaml

```diff
@@ -8,7 +8,7 @@ extra-deps:
   - postgresql-error-codes-1.0.1
   - tomland-1.3.3.1
   - validation-selective-0.1.0.1
-  - git: git@github.com:haskell-servant/servant.git
+  - git: https://github.com/haskell-servant/servant
```

Comment:
just out of curiosity, why do we need this change?

Reply:
CI failed to fetch the code otherwise. I wrote this long ago and can't remember exactly why, though, but here's a screenshot of the actions (and the solution):

![image](https://user-images.githubusercontent.com/12649588/210321624-c5839008-ec71-4348-b586-52ed0a9ccfb4.png)

The logs have expired so I can't inspect them.